### PR TITLE
v0.96.6 - Limited Zeus as lobby parameter

### PR DESCRIPTION
**Pull request by veteran29**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes  |
| Needs wipe? | no  |
| Fixed issues | - |

### Description:
Adds limited zeus as lobby parameter, enabled by default as it was always so far. Allows to enable full Zeus without editing mission.sqm

### Content:
- [x] Limited Zeus lobby paramter
- [x] Handling of zeus parameter
- [x] Added hint suggesting to change default of LoadSaveParams param if editing params via files (common issue)

### Successfully tested on:
- [x] Local MP
- [ ] Dedicated MP

### Compatibility checked with:
* None

### CODE REVIEW

File: Missionframework/scripts/server/init_server.sqf [hunk 1]
```diff
@@ -126,3 +126,26 @@ execVM "scripts\server\offloading\group_diag.sqf";
...
 };
+
+if (KP_liberation_limited_zeus) then {
+    zm1 setVariable ["Addons", 0, true];
+    removeAllCuratorAddons zm1;
+
+    zm1 setCuratorCoef ["edit", -1e8];
+    zm1 setCuratorCoef ["place", -1e8];
+    zm1 setCuratorCoef ["synchronize", 0];
+    zm1 setCuratorCoef ["delete", 0];
```
Comment (veteran29) [inline Missionframework/scripts/server/init_server.sqf]: Not sure about this (delete), it was always set to zero but it seems like an oversight to me. Limited curator should not be allowed to destroy objects IMO. Synchronize needs to stay as it allows to group up units.

Comment (veteran29) [reply Missionframework/scripts/server/init_server.sqf]: Yeah, that s why I think it's an oversight, however it was like that for years and no one complained 🤔

Comment (Wyqer) [reply Missionframework/scripts/server/init_server.sqf]: It's needed for being able to "recycle" the helipads, which can't be targeted in normal view for displaying an action menu entry. And to maybe delete own AI units etc. if they aren't needed anymore, far far away or the AI got stuck etc.
Concerning enemy units: These aren't added to Zeus and therefore won't be able to be deleted by the commander.

File: Missionframework/scripts/server/init_server.sqf [hunk 2]
```diff
@@ -126,3 +126,26 @@ execVM "scripts\server\offloading\group_diag.sqf";
 if (KP_liberation_restart > 0) then {
     execVM "scripts\server\game\server_restart.sqf";
 };
+
+if (KP_liberation_limited_zeus) then {
+    zm1 setVariable ["Addons", 0, true];
```
Comment (Wyqer) [inline Missionframework/scripts/server/init_server.sqf]: Nice, didn't know that this is just stored as `Addons` variable. Guess looked it up in the module functions?
Great finding. 👍

File: Missionframework/stringtable.xml [hunk 3]
```diff
@@ -2422,6 +2422,9 @@
             <Portuguese>Remover carga do veículo requisitado</Portuguese>
             <Chinese>移除載具上的物品</Chinese>
         </Key>
+        <Key ID="STR_PARAM_LIMITEDZEUS">
+            <Original>Limited Zeus interface</Original>
```
Comment (Wyqer) [inline Missionframework/stringtable.xml]:
```suggestion
            <Original>Limited Zeus interface</Original>
            <German>Eingeschränkte Zeus Funktionen</German>
```

File: Missionframework/scripts/server/init_server.sqf (hunk 2 above)
Comment (veteran29) [reply Missionframework/scripts/server/init_server.sqf]: Yeah it's from `moduleCurator`, also the same way is used in `virtual` module of 0.97

**Comment by veteran29** [comment]
@KillahPotatoes/kplib-testers (or anyone watching this), I would be grateful for test on a dedicated server. Expecially testing JIP and slot swapping etc.

You can get built PBOs from CircleCI [artifacts](https://circleci.com/gh/KillahPotatoes/KP-Liberation/1137#artifacts/containers/0)

**Comment by zharf** [comment]
Looks good to me, pretty sure JIP shouldn't be a problem but I can do some tests on that

**Pull request merged**